CRITICAL - 2022-11-23 14:28:45 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:34:16 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:39:41 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:42:03 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:47:32 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:47:33 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:49:21 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:49:22 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:50:16 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:50:17 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:50:18 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 14:50:19 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-11-23 14:58:54 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-23 14:58:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Rules.php(116): CodeIgniter\Database\BaseBuilder->get()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(277): CodeIgniter\Validation\Rules->is_not_unique('RockyRock', 'username', Array, NULL)
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(151): CodeIgniter\Validation\Validation->processRules('username', 'username', 'RockyRock', Array, Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(77): CodeIgniter\Controller->validate(Array)
#7 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(31): App\Controllers\Auth->login_action()
#8 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#9 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-23 15:00:31 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:32 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:33 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:34 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:35 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:36 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:37 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:38 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:39 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:40 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:41 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:43 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:46 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-11-23 15:00:47 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-11-23 15:01:24 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-23 15:01:24 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Rules.php(116): CodeIgniter\Database\BaseBuilder->get()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(277): CodeIgniter\Validation\Rules->is_not_unique('RockyRock', 'username', Array, NULL)
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(151): CodeIgniter\Validation\Validation->processRules('username', 'username', 'RockyRock', Array, Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(77): CodeIgniter\Controller->validate(Array)
#7 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(31): App\Controllers\Auth->login_action()
#8 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#9 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-23 15:01:42 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-23 15:01:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Rules.php(116): CodeIgniter\Database\BaseBuilder->get()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(277): CodeIgniter\Validation\Rules->is_not_unique('RockyRock', 'username', Array, NULL)
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(151): CodeIgniter\Validation\Validation->processRules('username', 'username', 'RockyRock', Array, Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(77): CodeIgniter\Controller->validate(Array)
#7 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(31): App\Controllers\Auth->login_action()
#8 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#9 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-23 15:01:54 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-11-23 15:01:54 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-11-23 15:01:54 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-11-23 15:01:55 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-11-23 15:01:56 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-11-23 15:01:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-23 15:01:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Rules.php(116): CodeIgniter\Database\BaseBuilder->get()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(277): CodeIgniter\Validation\Rules->is_not_unique('RockyRock', 'username', Array, NULL)
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(151): CodeIgniter\Validation\Validation->processRules('username', 'username', 'RockyRock', Array, Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(77): CodeIgniter\Controller->validate(Array)
#7 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(31): App\Controllers\Auth->login_action()
#8 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#9 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-23 15:02:00 --> Undefined index: HTTP_USER_AGENT
#0 /home/fallenxt/public_html/fallenx/app/Views/Auth/login.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/home/fallenxt/...', 32, Array)
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Auth/login', Array, true)
#4 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(37): view('Auth/login', Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#6 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/fallenxt/public_html/fallenx/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-11-23 15:02:15 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-23 15:02:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Rules.php(116): CodeIgniter\Database\BaseBuilder->get()
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(277): CodeIgniter\Validation\Rules->is_not_unique('RockyRock', 'username', Array, NULL)
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Validation/Validation.php(151): CodeIgniter\Validation\Validation->processRules('username', 'username', 'RockyRock', Array, Array)
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(77): CodeIgniter\Controller->validate(Array)
#7 /home/fallenxt/public_html/fallenx/app/Controllers/Auth.php(31): App\Controllers\Auth->login_action()
#8 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\Auth->login()
#9 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-23 15:24:48 --> mysqli_connect(): (HY000/1045): Access denied for user 'fallenx1_jangra'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/fallenx/app/Controllers/conn.php(8): mysqli_connect('localhost', 'fallenx1_jangra', 'fallenx1_jangra', 'fallenx1_jangra')
#2 /home/fallenxt/public_html/fallenx/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/fallenx/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/fallenx/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
